app/indexers/rights_metadata_indexer.rb

```diff
@@ -100,10 +42,34 @@ def to_solr
     'http://opendatacommons.org/licenses/odbl/1.0/' => 'odc-odbl'
   }.freeze
 
+  def rights_descriptions
+    return 'controlled digital lending' if cocina.access.controlledDigitalLending
+
+    basic_access = build_basic_access(cocina.access)
+    files = Array(cocina.structural.contains).flat_map { |fs| fs.structural.contains }
```

Comment:
`Array(fs.structural.contains)`?

Reply:
cast this to an array.  If it's `nil` it becomes an empty array.  If it's already an array return itself.

Reply:
Ah, very good idea. Done.